packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -260,7 +263,77 @@ const AccountRecoveryResetPassword = ({
       logViewEvent(viewName, 'verification.success');
 
       // FOLLOW-UP: Functionality not yet available.
-      await broker.invokeBrokerMethod('afterCompleteResetPassword', account);
+      // await broker.invokeBrokerMethod('afterCompleteResetPassword', account);
+
+      switch (integration.type) {
```

Comment:
Thanks for breaking it out like this. It is very clear what is going on with different integrations. That being said...maybe we can simplify this and only have one action (user gets `Password reset succesffully. Please sign-in again`). It is a product question for sure, but maybe the simplification of this will help the refactor.